Cargo.toml

```diff
@@ -21,13 +21,12 @@ travis-ci = { repository = "slide-rs/specs" }
 crossbeam = "0.3.0"
 derivative = "1"
 fnv = "1.0"
-hibitset = { version = "0.4.1", features = ["parallel"] }
+hibitset = { version = "0.4.1", default-features = false }
 mopa = "0.2"
 shred = "0.6.0"
 shrev = "0.8.0"
-shred-derive = "0.4"
```

Comment:
I think you accidentally deleted this line.

Reply:
Oops, restored